bitnami/magento/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: magento
-version: 5.0.3
+version: 5.0.4
```

Comment:
Maybe we should increment the chart version in the `Chart.yml` in a minor, according to [semver](http://semver.org/):

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.